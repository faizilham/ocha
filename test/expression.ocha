print("OK");                        // OK
print(0);                           // 0
print(nil);                         // nil
print(true);                        // true
print(false);                       // false

print(1 + 1);                       // 2
print(2 * 3);                       // 6
print(2 - 6);                       // -4
print(1.0 / -2);                    // -0.5

print(1 < 2);                       // true
print(1 > 10);                      // false
print(2 <= 2);                      // true
print(!(1 >= 10));                  // true
print("test" == "test");            // true
print(1 != nil);                    // true
print(1 < 2 ? "a" : "b");           // a
