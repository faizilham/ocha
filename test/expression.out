OK
0
nil
true
false
2
6
-4
-0.5
true
false
true
true
true
true
a
